mock/breez-sdk/src/lib.rs

```diff
@@ -648,9 +696,23 @@ pub async fn parse(input: &str) -> Result<InputType> {
         return Ok(Bolt11 { invoice });
     }
 
-    // If the input is not a Bolt11, we just assume it is a LNURL-withdraw request.
-    // This is being used for topups, LNURL-pay is less used.
     // Without requesting the server, it is not possible to know whether an LNURL string is a pay or withdraw request
+    // So instead we interpret the string 'lnurlp' as if it was an LNUrL-pay string
+    if input == "lnurlp" {
+        println!("Returning LnUrlPay");
```

Comment:
```suggestion
        debug!("Returning LnUrlPay");
```

Reply:
Oh sorry, that was only for development purposes. I'm gonna remove it, I don't think there's much value in writing this to the log.

Reply:
94f3cc6597d07becb26f658c5cb93ff95fcef133